spec/controllers/v0/sessions_controller_spec.rb

```diff
@@ -36,18 +36,19 @@
         post :saml_callback
         expect(User.find(uuid).loa).to eq(highest: LOA::THREE, current: LOA::THREE)
       end
-      it 'redirects to an auth failure page if saml_response is invalid' do
-        allow(OneLogin::RubySaml::Response).to receive(:new).and_return(invalid_saml_response)
-
-        expect(Rails.logger).to receive(:error).exactly(1).times
-        expect(post(:saml_callback)).to redirect_to(SAML_CONFIG['relay'] + '?auth=fail')
-        expect(response).to have_http_status(:found)
-      end
       it 'creates a valid session and user' do
         post :saml_callback
         expect(User.find(uuid)).to_not be_nil
         expect(User.find(uuid).attributes).to eq(User.from_merged_attrs(loa1_user, loa3_user).attributes)
       end
+      context ' when SAMLResponse is invalid' do
```

Comment:
This is a bit of an unrelated refactor.  I thought the SAML Response being invalid warranted its own context so it may be reused.